### PR TITLE
Skip TLS check for the session

### PR DESCRIPTION
**Pull request by vishpat**
@primalmotion

### CODE REVIEW

File: _posts/2016-02-20-Go Bambo Go.md [hunk 1]
```diff
@@ -116,7 +116,8 @@ func main() {
 
 	session, root := vspk.NewSession("csproot", "csproot", "csp",
         "https://api.nuagenetworks.net:8443")
-
+        
+        session.SetInsecureSkipVerify(true)
```
Comment (primalmotion) [inline _posts/2016-02-20-Go Bambo Go.md]: Alignment

File: _posts/2016-02-20-Go Bambo Go.md [hunk 2]
```diff
@@ -143,8 +144,9 @@ func main() {
 	session, root := vspk.NewSession("csproot", "csproot", "csp",
         "https://api.nuagenetworks.net:8443")
 
+	session.SetInsecureSkipVerify(true)
 	session.Start()
-
+
```
Comment (primalmotion) [inline _posts/2016-02-20-Go Bambo Go.md]: Whitespaces

**Comment by primalmotion** [comment]
Thanks!

**Comment by vishpat** [comment]
@primalmotion  Have addressed the review comments.

**Comment by primalmotion** [comment]
Merged, Thanks!

**Pull request merged**